client/src/actions/index.ts

```diff
@@ -179,6 +179,30 @@ async function fetchGeneInfo(
   return response;
 }
 
+interface CellTypeInfoAPI {
+  name: string;
+  summary: string;
+}
+/**
+ * Fetch cell type summary information
+ * @param cell human-readable name of cell type
+ */
+async function fetchCellTypeInfo(
+  cell: string,
+  dispatch: AppDispatch
+): Promise<CellTypeInfoAPI | undefined> {
+  try {
+    return await fetchJson<CellTypeInfoAPI>(`cellinfo?cell=${cell}`);
+  } catch (error) {
+    dispatchNetworkErrorMessageToUser("Failed to fetch cell type information.");
+    dispatch({
+      type: "request cell info error",
```

Comment:
@tihuan - this will be dispatched on error thus populating the error field and then display the error component to the user

Reply:
Ah fabulous 💡 Thanks for adding this!